codegen/src/test/resources/software/amazon/awssdk/codegen/poet/client/c2j/fine-grained-auth-legacy-trait/service-2.json

```diff
@@ -0,0 +1,1012 @@
+      },
+      "input":{"shape":"OperationWithExplicitPayloadBlobInput"},
+      "output":{"shape":"OperationWithExplicitPayloadBlobInput"}
+    },
+    "OperationWithExplicitPayloadStructure":{
+      "name":"OperationWithExplicitPayloadStructure",
+      "http":{
+        "method":"POST",
+        "requestUri":"/2016-03-11/operationWithExplicitPayloadStructure"
+      },
+      "input":{"shape":"OperationWithExplicitPayloadStructureInput"},
+      "output":{"shape":"OperationWithExplicitPayloadStructureInput"}
+    },
+    "OperationWithGreedyLabel":{
+      "name":"OperationWithGreedyLabel",
+      "http":{
+        "method":"POST",
+        "requestUri":"/2016-03-11/operationWithGreedyLabel/{NonGreedyPathParam}/{GreedyPathParam+}"
+      },
+      "input":{"shape":"OperationWithGreedyLabelInput"}
+    },
+    "OperationWithModeledContentType":{
+      "name":"OperationWithModeledContentType",
+      "http":{
+        "method":"POST",
+        "requestUri":"/2016-03-11/operationWithModeledContentType"
+      },
+      "input":{"shape":"OperationWithModeledContentTypeInput"}
+    },
+    "OperationWithNoInputOrOutput":{
+      "name":"OperationWithNoInputOrOutput",
+      "http":{
+        "method":"POST",
+        "requestUri":"/2016-03-11/operationWithNoInputOrOutput"
+      }
+    },
+    "PaginatedOperationWithResultKey": {
+      "name": "PaginatedOperationWithResultKey",
+      "http": {
+        "method": "POST",
+        "requestUri": "/"
+      },
+      "input": {
+        "shape": "PaginatedOperationWithResultKeyRequest"
+      },
+      "output": {
+        "shape": "PaginatedOperationWithResultKeyResponse"
+      },
+      "documentation": "Some paginated operation with result_key in paginators.json file"
+    },
+    "QueryParamWithoutValue":{
+      "name":"QueryParamWithoutValue",
+      "http":{
+        "method":"POST",
+        "requestUri":"/2016-03-11/queryParamWithoutValue?param"
+      },
+      "input":{"shape":"QueryParamWithoutValueInput"}
+    },
+    "StreamingInputOperationWithRequiredChecksum":{
+      "name":"OperationWithRequiredChecksum",
+      "http":{
+        "method":"POST",
+        "requestUri":"/2016-03-11/allTypes"
+      },
+      "input":{"shape":"StructureWithStreamingMember"},
+      "output":{"shape":"AllTypesStructure"},
+      "httpChecksumRequired": true
+    },
+    "OperationWithRequiredChecksum":{
+      "name":"OperationWithRequiredChecksum",
+      "http":{
+        "method":"POST",
+        "requestUri":"/2016-03-11/allTypes"
+      },
+      "input":{"shape":"AllTypesStructure"},
+      "output":{"shape":"AllTypesStructure"},
+      "httpChecksumRequired": true
+    },
+    "OperationWithNoneAuthType":{
+      "name":"OperationWithNoneAuthType",
+      "http":{
+        "method":"POST",
+        "requestUri":"/2016-03-11/allTypes"
+      },
+      "input":{"shape":"AllTypesStructure"},
+      "output":{"shape":"AllTypesStructure"},
+      "authtype": "none"
```

Comment:
nit: this is a huge file for just this test case. could you use a simpler model for this?